### ISSUE TITLE
[Issue] Request support for CJK characters(Chinese, Japanese, and Korean languages) 

### ISSUE THREAD

**Issue by Wensupu-Yang** [bkyle/obsidian-vault-statistics-plugin#23]
Thank you for creating this plugin-in. I've been always want to know how many notes and words have I written. 

However, I noticed, it doesn't work with CJK characters. As shown in the screenshots.

I'd be really appreciated it if it can be fixed in future updates. 
<img width="368" alt="Screenshot 2023-01-06 at 4 14 17 PM" src="https://user-images.githubusercontent.com/26530267/211053792-1618dca4-4661-404a-b099-fa9b61a56f0c.png">

<img width="370" alt="Screenshot 2023-01-06 at 4 14 36 PM" src="https://user-images.githubusercontent.com/26530267/211053785-6d499f54-9bcb-4e3d-873a-90dafc73d62c.png">

<img width="368" alt="Screenshot 2023-01-06 at 4 19 59 PM" src="https://user-images.githubusercontent.com/26530267/211053765-4b914e46-9bf6-4281-9068-1217a183407d.png">

**Comment by HauxHe** [comment]
I have the same issues.

**Comment by groepl** [comment]
Yes, that will be a brilliant idea. But I have no idea, how to build that feature? Any hint from you?

**Comment by chriszhuliping** [comment]
> Yes, that will be a brilliant idea. But I have no idea, how to build that feature? Any hint from you?

For CJK, we count characters(without characters like space or punctuation) when we doing the word count.
For example, when we say hello in Chinese, we type 你好. so in English, it is a word, but in CJK we count it as 2 words(actually 2 characters).
I'm not sure if I explained that well, hope that it will help you!
